src/modules/addPlace/addPlace.view.js

```diff
@@ -0,0 +1,98 @@
+import React, { Component } from 'react';
+import { Flex, InputItem, List, Button, WhiteSpace, WingBlank, TextareaItem, NoticeBar } from 'antd-mobile';
+import { observable, reaction, observe } from 'mobx';
+import { inject, observer } from 'mobx-react';
+import { Bind } from 'lodash-decorators';
+import DeviceService from '../../services/DeviceService';
+import { Select } from 'antd';
+import 'antd/dist/antd.css';
+import OfficeService from '../../services/OfficeService';
+import PlaceService from '../../services/PlaceService';
+
+@inject('officesStore')
+@observer
+class AddPlace extends Component {
+  @observable name = '';
+  @observable selectedPlaceId = '';
+
+  @observable errorMsg;
+
+  @Bind
+  onChangeName(value) {
+    this.name = value;
+  }
+
+  @Bind
+  onChangeOffice(value) {
+    this.selectedPlaceId = value;
+  }
+
+  @Bind
+  sendToDatabase() {
+    if (this.name.length != 0 &&
+      this.selectedPlaceId != 0) {
+      const data = {
+        name: this.name,
+        officeId: this.selectedPlaceId
+      }
+      PlaceService.postNewPlace(data);
+      this.errorMsg = 'New place added';
```

Comment:
Wow, that's not an error :D maybe we can use better name?

Reply:
Or even better, if place is added successfully, redirect user to places list, or this place details view